vignettes/release_strategy.Rmd

```diff
@@ -41,37 +41,29 @@ updates via the community meetings, especially any anticipated breaking changes.
 
 ## Quarterly Release
 
-A package release is done in five parts:
+A package release is done in four parts:
 
-1) Create a Pull Request from `devel` into the `main` branch.  Issues identified in this Pull Request should have work done in separate branches and merged once again into `devel`.
-1) Verify that all CI/CD checks are passing for the `devel` into the `main` Pull Request, merge and then bundle up and send off to CRAN. See the [chapter](https://r-pkgs.org/release.html#decide-the-release-type) in R Packages for more details.
+1) Verify that all CI/CD checks and then bundle up and send off to CRAN. See the [chapter](https://r-pkgs.org/release.html#decide-the-release-type) in R Packages for more details.
 1) If CRAN asks for modifications, repeat steps 1-2 as necessary.
 1) Once the package is accepted and available on CRAN, a [GitHub action](https://github.com/pharmaverse/admiral/actions/workflows/pages/pages-build-deployment) is set up to rebuild the `{admiral}` website with all the updates for this release.
 1) Use the release button on GitHub to "release" the package onto GitHub.  This release onto Github archives the version of code within the `main` branch, attaches the News/Changelog file, bundles the code into a `tar.gz` file and makes a validation report via the GitHub action `validation` from [insightsengineering/validatoR](https://github.com/insightsengineering/thevalidatoR).  Please see past [admiral releases](https://github.com/pharmaverse/admiral/releases) for reference and the [Releasing to Github](release_strategy.html#releasing-to-github) section for more details.
-1) Any issues fixed in the `main` branches should be merged back into `devel`. 
```

Comment:
Do we need a PR to `main` which adds the `.9000` again or should this be done in the first feature branch?

Reply:
When we make releases, we should follow the checklist issue created from `usethis::use_release_issue()`. The last thing to do there is bump the version to `xxx.9000`. There is no need to wait until the first PR to increase it (although this is certainly something i have done in the past, and either way works perfectly well. But good to stick with standard practices when we can)

Reply:
Should we use `use_release_issue()` and follow the list created by the call or should we follow our list? Unfortunately I couldn't find an example of the list created by `use_release_issue()` in the usethis documentation. Most likely there are many overlaps. I think we should have just one list to follow.